seqr/utils/elasticsearch/constants.py

```diff
@@ -334,8 +335,11 @@ def get_prediction_response_key(key):
     'end': {},
     'start': {},
     'num_exon': {},
-    'geneIds': {'response_key': 'geneIds'},
+    'geneIds': {'response_key': 'geneIds', 'format_value': list},
     'defragged': {'format_value': bool},
+    'prev_call': {'format_value': bool},
```

Comment:
Would it be better to use `prev_identical`?

Reply:
no because thats not whats in the index?